.github/workflows/make-apk.yml

```diff
@@ -7,14 +7,24 @@ on:
 
 jobs:
   Build-Apk:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-20.04 # That is the latest OS version available today
     steps:
-    - name: Install external dependencies
+    - name: Cache fontforge and extra dependencies
+      id: cache_localbin
+      uses: actions/cache@v2
+      with:
+        path: /usr/local/bin
+        key: usr-local-bin
+    - name: Install latest FontForge version (using AppImage)
+      if: steps.cache_localbin.outputs.cache-hit != 'true'
       run: |
-        # Dirty: Get a more recent version of fontforge
-        sudo sed -i 's/focal/impish/g' /etc/apt/sources.list
-        sudo apt-get update -y
-        sudo apt-get install -y fontforge
+        # Get most recent version of FontForge
+        # Using AppImage there is no dependecy problem, it is the latest version and it's easier to cache
+        # -------
+        cd /usr/local/bin
+        sudo wget -c -N https://github.com/fontforge/fontforge/releases/download/20220308/FontForge-2022-03-08-582bd41-x86_64.AppImage
+        sudo chmod +x ./FontForge-2022-03-08-582bd41-x86_64.AppImage
+        sudo ln --symbolic --force /usr/local/bin/FontForge-2022-03-08-582bd41-x86_64.AppImage /usr/local/bin/fontforge
```

Comment:
This can be a `mv`.

Reply:
Only thing though, with the `mv` method, by renaming the wget target, wget will redownload the target again in the next run. and that would also make the caching process more complex, we would end up with a bunch of _if_ statements to avoid redownload fontforge again.
Leaving the dowloaded file the original name, the cache will populate the file before wget gets executed, and then wget doesn't need to download anything when we have cache.

Reply:
This if statement is already here: `if: steps.cache_localbin.outputs.cache-hit != 'true'`. If this line doesn't do anything interesting, it should be removed.

Reply:
I'm not fully confident that `if: steps.cache_localbin.outputs.cache-hit != 'true'` is doing all the work alone, so I rather eliminate `if: steps.cache_localbin.outputs.cache-hit != 'true'` and rely only on the `wget` behaviour of not downloading a file again if there is no need.

Reply:
removed the redundant lines